packages/next-common/components/changeableConnectedSigner.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import styled from "styled-components";
+import Account from "./account";
+import EmptyAccount from "./emptyAccount";
+import { GreyPanel } from "./styled/containers/greyPanel";
+import { useSetConnectedWallet } from "next-common/context/connectedWallet";
+import { CACHE_KEY } from "next-common/utils/constants";
+
+const Wrapper = styled(GreyPanel)`
+  padding: 12px 16px;
+  justify-content: space-between;
+`;
+
+export default function ChangeableConnectedSigner({ signerAccount }) {
+  const setConnectedWallet = useSetConnectedWallet();
+
+  const reSelectWallet = () => {
+    setConnectedWallet();
+    localStorage.removeItem(CACHE_KEY.lastLoginAddress);
```

Comment:
I think we can create a function `disconnectAddress` in the `ConnectedWalletProvider` component which will be a better encapsulation.